src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -9,13 +9,14 @@
 using System.Linq;
 using System.Net.Cache;
 using System.Net.Http;
-using System.Net.Sockets;
 using System.Net.Test.Common;
 using System.Runtime.Serialization.Formatters.Binary;
+using System.Security.Authentication;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.DotNet.RemoteExecutor;
+using Newtonsoft.Json;
```

Comment:
There are some build failures with this PR since Newtonsoft is not automatically brought into the build process. If you want to use this, you'll need to modify the CSPROJ for the tests to bring in the NuGet package.

However, this might be a good opportunity to use the new .NET Core 3.0 System.Json API instead of Newtonsoft.

Reply:
Switched to System.Text.Json.